content/_index.html

```diff
@@ -42,6 +42,35 @@ <h1 class="headline1 font-medium">
   </div>
 </section>
 
+<section class="glue-page gpb-8">
+  <a id="get-started"></a>
```

Comment:
Can this `<a>` element with empty content be removed?

Reply:
+1, good spot